TotallyNotMonopolyAPI/Model/Player.cs

```diff
@@ -11,5 +11,6 @@ public class Player
         public int Id { get; set; }
         public string Name { get; set; }
         public int Money { get; set; } = 1500;
+        public List<Property> properties { get; set; }
```

Comment:
C# properties should by uppercase.

public List<Property> Properties { get; set; }